crates/http/src/routes.rs

```diff
@@ -6,6 +6,7 @@ use anyhow::{bail, Result};
 use http::Uri;
 use indexmap::IndexMap;
 use spin_manifest::{Application, CoreComponent};
+use std::fmt;
```

Comment:
nit: could you please merge the two `std` imports?

Reply:
Done.